samples/chat/README.md

```diff
@@ -88,7 +86,29 @@ Please refer to the root level [README](../../README.md#requirements) for prereq
     Who won the SuperBowl in 2014?
```

Comment:
I assume these samples need to be udpated with the new HTTP request trigger format?

Reply:
Updated